src/tests/system/tests/test_ipa.py

```diff
@@ -220,6 +220,80 @@ def test_ipa__hostpublickeys_with_non_default_port(client: Client, ipa: IPA, pub
         )
 
 
+@pytest.mark.importance("medium")
+@pytest.mark.integration
+@pytest.mark.topology(KnownTopology.IPA)
```

Comment:
Just because the IPAUser class already had functions to add a certificate. This is not necessarily a test we need to run on each provider, I don't think.

Reply:
Just because the IPAUser class already had functions to add a certificate. This is not necessarily a test we need to run on each provider, I don't think.